packager/app/test/testdata/audio-video-with-codec-switching-encryption-trick-play/output.mpd

```diff
@@ -2,28 +2,14 @@
 <!--Generated with https://github.com/google/shaka-packager version <tag>-<hash>-<test>-->
```

Comment:
As explained in the previous PR, the test has non-deterministic behavior. Since this test is irrelevant, can you revert this file?